src/AutorestSwift/JsonRpc/Client.swift

```diff
@@ -30,70 +30,82 @@ import Foundation
 import NIO
 
 public final class ChannelClient {
-    public let group: MultiThreadedEventLoopGroup
+    private let group: MultiThreadedEventLoopGroup
     public let config: Config
-    private var channel: Channel?
-
-    public init(group: MultiThreadedEventLoopGroup, config: Config = Config()) {
+    private var context: ChannelHandlerContext?
+    private let processCallback: ProcessCallback
+    private let logger = FileLogger(withFileName: "autorest-swift-debug.log")
+    // TODO: get this value based on last response id from AutoRest
+    var id = 3
```

Comment:
What is this used for?

Reply:
This is the initial value for the request id when we send message (as a client ) to Autorest.  We should fix it to retrieve it programmically (this value should be the id of the last message we get from Autorest when we were acting as Server.

We need to increment this value (id of the request) for each request when we send message to Autorest.

Reply:
Wil open a new PR to address this. Don't want this PR to be so big.